modules/statics/src/base.ts

```diff
@@ -143,6 +143,10 @@ export enum CoinFeature {
    * This coin is deprecated
    */
   DEPRECATED = 'deprecated',
+  /**
+   * This coin is a dummy object meant to be a placeholder for an unsupported token
+   */
+  GENERIC = 'generic',
```

Comment:
This field looks too "generic" to be used for tokens on the base definitions.

Reply:
Since the verbage everywhere here is "nonstandard", maybe something closer to `TOKEN_NONSTANDARD`?

Reply:
I'm not sure I fully understand, but this seems like a work around rather than the needed solution.

Reply:
`nonstandard` is meant to describe which ERC or smart contract standard the token implements. Generic on the other hand is meant to be an internal descriptor; it has no indication of the smart contract contract standards and only is meant to indicate if a token object represents a specific token or a dummy (catch all) token. The need for this is due the fact that BitGo can "support" all NFTs without having to whitelabel each NFT that comes into a BitGo wallet.

Reply:
`GENERIC` here is still too wide of a name for all coins and tokens to represent a dummy token specifically. Needs a more focused name. Something like `TOKEN_GENERIC` feels closer to represent a more niche token feature of coin features.

Reply:
Especially since may coins don't have tokens.

Reply:
The needed solution is likely a larger scope service that is the single source of truth for tokens which is updated/expanded dynamically and automatically as opposed to tokens listed manually across our tech stack. So yes this is definitely more of a work around to allow us to show customers their transfers for NFTs even if the NFT is not "supported".

Reply:
Ah okay I see what you are saying. I will crate a follow up PR to use a more accurate name.

Reply:
Thanks, I appreciate it. It looks like `GENERIC_TOKEN_FEATURES` is used lower so maybe `GENERIC_TOKEN` would match that?

Reply:
PR to use more semantic naming https://github.com/BitGo/BitGoJS/pull/3033